index.js

```diff
@@ -99,6 +99,21 @@ client.on("message", async message => {
     'Outlook not so good',
     'Very doubtful'
   ];
+//!ping to run ping
+client.on('message', message => {
+if (message.author.bot) return;
+if (!message.content.startsWith('!')) return;
```

Comment:
This will trigger the Rhythm bot as well.

Reply:
ye my bad I forgot it should be queen ping. Here is the updated code

client.on('message', message => {
if (message.author.bot) return;
if (!message.content.startsWith('queen')) return;
if (message.content.startsWith('queen' + 'ping')) {
    message.channel.send({embed: {
        color: 0x2ed32e,
        fields: [{
            name: "Your Ping",
            value: "My Ping: " + Math.round(client.ws.ping) + ' ms'
      }
     ],
}
})
}})

Reply:
You also can't have a client.on() within another client.on()

Reply:
ye lets reject my pull request i'll resubmit it again lol